library/n2_protocol.cpp

```diff
@@ -144,6 +144,22 @@ void n2_response_info_free(struct n2_response_info *resp_info) {
 	delete resp_info;
 }
 
+struct n2_response_info *n2_response_info_create_from_error(struct dnet_cmd *cmd, struct n2_repliers *repliers,
+                                                            int err) {
+	auto response_holder = std::bind(repliers->on_reply_error, err, true);
```

Comment:
`std::bind(&n2_repliers::on_last_error, repliers, err)` or, please, add comment to `true`

Reply:
`std::bind(&n2_repliers::on_last_error, repliers, err)` is not good because `repliers` is just a pointer, and doing so we aren't prolongate `repliers->on_reply_error`'s lifetime.

Reply:
Comment is also superfluous, we don't comment params of each call.